### ISSUE TITLE
Heads up: Deprecations django.utils.translation.ugettext_lazy and django.conf.urls.url

### ISSUE THREAD

**Issue by hwalinga** [netzkolchose/django-computedfields#68]
There are a few functions used in this package that will be deprecated:

```
python3.9/site-packages/computedfields/models.py:85: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is
deprecated in favor of django.utils.translation.gettext_lazy().
    verbose_name_plural = _('Computed Fields Models') 
```

and 

```
ython3.9/site-packages/computedfields/admin.py:110: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favo
r of django.urls.re_path().      
    url(r'^computedfields/modelgraph/(?P<modelid>\d+)/$'
```

I know Django 4.0 is far away (about one year), but seeing the warnings all the time gets a bit old after some time. But just a heads up so that you are aware about this. 

Thanks for the work.

**Comment by jerch** [comment]
Thx for the heads up. I currently aim for stable support of LTS versions, thus the next API cleanup/transition is planned for v3.2 (coming around April 2021). By that time I will check for those warnings as well and whether they need workarounds for older versions.

If you care enough to get rid of those warnings earlier - feel free to create a PR already dealing with them. (In fact the `ugettext_lazy()` looks like a remnant from Python 2.x / Django 1.11 times).

**Comment by hwalinga** [comment]
Django 3.2 comes with one more: 

```
  /home/hielke/.cache/pypoetry/virtualenvs/demap-kqoNszLJ-py3.9/lib/python3.9/site-packages/django/apps/registry.py:91: RemovedInDjango41Warning: 'computedfields' defines default_app_config = 'computedfields.apps.ComputedfieldsConfig'. Django now detects this configuration automatically. You can remove default_app_config.
    app_config = AppConfig.create(entry)
```

If you want some help with it, you can ping me, but it should not be too difficult.

**Comment by jerch** [comment]
This might need a version switch in the code to also do the right thingy in lower version (not sure yet).

> If you want some help with it, you can ping me, but it should not be too difficult.

Sure thing, feel free to either create another PR or add it to #72. There I already started to apply the changes, sadly there are more changes to the ORM in 3.2 so this module does not work with it yet.

**Comment by jerch** [comment]
Fixed in #72.
